### PR TITLE
minor edits to ms

### PR DESCRIPTION
**Pull request by ckyriazis**
made a few edits throughout the ms...tried to be more consistent about sp names, added a sentence explaining our use of census size in the two pop case, etc.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
LGTM - I'll let @andrewkern merge because I'm not up to speed enough to follow the last bit.

**Comment by andrewkern** [comment]
these are fine but we now have to deal with the name/ID switch for pongo...

**Comment by jeromekelleher** [comment]
Looks like you need to rebase & fix some conflicts too @ckyriazis

**Comment by ckyriazis** [comment]
ok ill hold off on this until we decide what to do with pongo.

**Comment by andrewkern** [comment]
okay @ckyriazis we have clarity on this (https://github.com/popsim-consortium/manuscript/issues/2). We are sticking with PonPyg for the initial release. Can you fix up this PR and do a find replace to change everything to pygmaeus?

**Comment by ckyriazis** [comment]
sure sounds good. i'll take care of the dromel sims stuff here as well.

File: popSimManu.tex [hunk 1]
```diff
@@ -766,9 +772,7 @@ \subsection*{Workflow for analysis of simulated data}
 and specified taking 20 samples each from the Europe and Africa populations.
 For DroMel, runtimes under OutOfAfrica\_2L06 were prohibitively slow when simulating whole genomes.
 % RNG: Removed technical term "coalesce" here, since I don't think that's exactly the issue.
-For example, both chromosomes 3R and X failed to finish within 30 days of starting using $n = 50$.
-Runtimes using the identical sample size were 19, 21, and 12 days for chromosomes 2L, 2R, and 3L,
-respectively. For this reason, we chose to present only data from chromosome 2R for simulations
+For this reason, we chose to present only data from 3 Mb of chromosome 2R for simulations
 under OutOfAfrica\_2L06.
 % RNG: Not present in manuscript
 % For the generic IM simulations, we used the HomSap genome along with the
```
Comment (ckyriazis) [inline popSimManu.tex]: just realized that this got commented out - im going to update the PR to uncomment it since we have these results now

**Comment by ckyriazis** [comment]
ok that should do it.

File: popSimManu.tex [hunk 2]
```diff
@@ -362,18 +365,18 @@ \subsection*{The \stdpopsim Catalog}
...
 an African sample, as well as the out-of-Africa divergence
 and associated bottleneck model of \cite{li2006inferring} which jointly models African
 and European populations. For \emph{A. thaliana}, we implemented the
 model in \cite{durvasula2017african} inferred using \MSMC. This model includes
 a continuous change in population size over time, rather than pre-specified epochs of different
 population sizes. We have also implemented a two epoch and a three epoch model estimated from African
-samples of \emph{A. thaliana} in \cite{huber2018gene}. For \emph{P. abelii} we have implemented the two 
-species divergence model of \cite{locke2011comparative}, which models isolation with migration between \emph{P. abelii}
+samples of \emph{A. thaliana} in \cite{huber2018gene}. For \emph{P. pygmaeus} we have implemented the two
+species divergence model of \cite{locke2011comparative}, which models isolation with migration between \emph{P. pygmaeus}
```
Comment (jeromekelleher) [inline popSimManu.tex:377]: This should be abelii.

Comment (ckyriazis) [reply popSimManu.tex:377]: oops good catch

**Review by jeromekelleher — COMMENTED** [review]
LGTM, but need to change that one abelli back

**Comment by ckyriazis** [comment]
ok I fixed that abelii issue - this should be good to go.

**Pull request merged**